### PR TITLE
Adds Authentication and Return to Main Menu Feature

### PR DESCRIPTION
**Pull request by gaurnett**

### CODE REVIEW

File: sdk/custom/global/aog_open_main_menu.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+handler:
+    webhookHandler: aog_open_main_menu
+transitionToScene: aog_main_menu
```
Comment (sarahc7) [inline sdk/custom/global/aog_open_main_menu.yaml:3]: nit: new line

File: sdk/custom/intents/aog_open_main_menu.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+trainingPhrases:
+    - open AOG menu
+    - open AOG main menu
+    - restart AOG game
+    - open A O G menu
+    - open A O G main menu
+    - restart A O G game
```
Comment (sarahc7) [inline sdk/custom/intents/aog_open_main_menu.yaml:7]: We could add "main menu" as a training phrase so users don't have say "AOG" when they want to go back to the main menu.

File: sdk/custom/scenes/auth.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+onEnter:
+  webhookHandler: welcome
+conditionalEvents:
+- condition: user.accountLinkingStatus == "NOT_LINKED"
+  transitionToScene: auth_AccountLinking
+- condition: user.accountLinkingStatus == "LINKED"
+  transitionToScene: aog_main_menu
```
Comment (sarahc7) [inline sdk/custom/scenes/auth.yaml:7]: nit: new line

File: sdk/webhooks/ActionsOnGoogleFulfillment/config.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+exports.keys = {
+    clientID: '754184378768-t93qucvp0cp0udgp4dtddm21l05ce67b.apps.googleusercontent.com'
+}
```
Comment (sarahc7) [inline sdk/webhooks/ActionsOnGoogleFulfillment/config.js:3]: nit: new line

**Review by sarahc7 — APPROVED** [review]
Looks good - I have a few nits and a suggestion.

File: public/js/scene.js [hunk 5]
```diff
@@ -22,6 +22,11 @@ export class Scene {
         this.game.appendChild(this.menu.getMenu());
     }
 
+    aogOpenMainMenu() {
+        this.game.innerHTML = "";
```
Comment (Arboghast) [inline public/js/scene.js:26]: NIT: Change this to the RemoveChild method iteratively. Setting innerHTML like that has potential to cause issues with the rest of the app.

**Pull request merged**